documentation/asciidoc/topics/ref_hotrod_java_tutorials.adoc

```diff
@@ -26,6 +26,9 @@
 |link:{repository}/infinispan-remote/encoding[Encoding]
 |Demonstrates how encoding of caches work.
 
+|link:{repository}/infinispan-remote/programmatic-protostream-marshalling[Custom marshalling]
+|Demonstrates how custom marshalling works.
```

Comment:
I think the description should be update to reflect that this relates to the programmatic creation of Protostream schema and marshallers. We need to make it clear to users that this should only be done in exceptional circumstances and Protostream marshallers should be generated unless there's a use-case where this is not possible.

Reply:
I updated the comment